libtiny_tui/src/msg_area/line.rs

```diff
@@ -149,8 +149,12 @@ impl Line {
         self.len_chars += 1;
     }
 
-    /// How many lines does this take when rendered? O(n) where n = number of
-    /// split positions in the line (i.e. whitespaces).
+    pub(crate) fn force_recalculation(&mut self) {
```

Comment:
Not sure if this actually needs to be `pub(crate)`. `pub` might be the right choice, but I'm not sure if matters.